core/src/main/java/nl/nn/adapterframework/align/ToXml.java

```diff
@@ -209,9 +209,9 @@ public void handleElement(XSElementDeclaration elementDeclaration, N node) throw
 					XSAttributeUse attributeUse=(XSAttributeUse)attributeUses.item(i);
 					//if (DEBUG) log.debug("startElement ["+localName+"] attributeUse ["+ToStringBuilder.reflectionToString(attributeUse)+"]");
 					XSAttributeDeclaration attributeDeclaration=attributeUse.getAttrDeclaration();
-					if (DEBUG) log.debug("node ["+name+"] attributeDeclaration ["+ToStringBuilder.reflectionToString(attributeDeclaration)+"]");
+					//if (DEBUG) log.debug("node ["+name+"] attributeDeclaration ["+ToStringBuilder.reflectionToString(attributeDeclaration)+"]");
 					XSSimpleTypeDefinition attTypeDefinition=attributeDeclaration.getTypeDefinition();
-					if (DEBUG) log.debug("node ["+name+"] attTypeDefinition ["+ToStringBuilder.reflectionToString(attTypeDefinition)+"]");
+					//if (DEBUG) log.debug("node ["+name+"] attTypeDefinition ["+ToStringBuilder.reflectionToString(attTypeDefinition)+"]");
 					String attName=attributeDeclaration.getName();
```

Comment:
Als het dan toch niet altijd nodig is, vind ik het persoonlijk passend om er trace logging van te maken.
Die kan je dan lokaal aanzetten en zelfs naar je system.out laten printen voor snellere debugging...